ui/test-setup-script.js

```diff
@@ -1,9 +1,14 @@
 global.beforeEach(() => {
+    const appRoot = document.createElement('div');
+    appRoot.setAttribute('id', 'root');
+    document.body.appendChild(appRoot);
```

Comment:
For the `attachTo` to make the PopupTrigger work.

Reply:
🎉🍾